.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "0.1.0",
```

Comment:
How about a new PR that adds these files to the `.gitignore`, s.t. you don't need to worry about it in the future?